README.md

```diff
@@ -207,7 +207,7 @@ export default class CounterApp {
 
 #### Decorators
 
-The `@connect` decorator lets you create smart components less verbosely:
+The `@connect` decorator lets you create smart components more succinctly:
```

Comment:
Strange word `succinctly`.

I think something like `... smart components in a more compact way` would be better.